<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceDataManagePage</class>
 <widget class="QWidget" name="MaintainceDataManagePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>运行里程</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>牵引能耗</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>空压机1工作时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>空压机2工作时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>记录开始时刻</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_6">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>记录值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_time_mileage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2000-01-01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_time_energy">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2000-01-01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_time_airPress1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2000-01-01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_time_airPress2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2000-01-01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_value_mileage">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0.0km</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_value_energy">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>240</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0kwh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_value_airPress1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>310</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0kwh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_value_airPress2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87));
border:0px solid;
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>0kwh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum9">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum0">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNumC">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>510</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(50,50,50,255),stop:1 rgba(240,240,240,255));</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
